pykern/pkcli/fmt.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+u"""Wrapper for Python formatter (currently, ``black``) to update and to validate a repository.
+
+:copyright: Copyright (c) 2015 RadiaSoft LLC.  All Rights Reserved.
+:license: http://www.apache.org/licenses/LICENSE-2.0.html
+"""
+from pykern.pkdebug import pkdp
+import pykern.pksubprocess
+import pykern.pkunit
+
+def run(path):
+    '''
+        Run black formatter on path
+
+        Args:
+            path - string path to file or dir
+    '''
+    pykern.pksubprocess.check_call_with_signals(['black', f'{path}'])
```

Comment:
Black is unnecessarily noisy and emojiful which doesn't necessarily render in all environments (log files) so I think I would prefer `--quiet`. If people start complaining about that, we can add a configuration parameter.

Reply:
I think you just coined a new term in "emojiful"